### PR TITLE
update Dockerfile

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
update dockerfil

### CODE REVIEW

**Review by AdoAdoAdo — APPROVED** [review]

**Review by liviuf-ancas — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -101,3 +101,14 @@ The rest of endpoints remain the same.
 The faucet feature can be activated and users calling an endpoint will be able to perform requests that send a given amount of tokens to a specified address.
 
 In order to use it, first set the `FaucetValue` from `config.toml` to a value higher than `0`. This will activate the feature. Then, provide a `walletKey.pem` file near `config.toml` file. This will make the `/transaction/send-user-funds` endpoint available.
+
+
+## built docker image
```
[the remark this answers is not in the document]
Comment (miiu96) [reply README.md:106]: done

File: .github/workflows/deploy-docker.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      - name: Build image
+        run: |
+          TAG_VERSION=${{ steps.get_version.outputs.VERSION }}
+          cd ${GITHUB_WORKSPACE} && docker build -t "${REGISTRY_HOSTNAME}/${IMAGE_NODE}:${TAG_VERSION}" -f ./docker/Dockerfile .
+      - name: Push image
+        run: |
+          docker login --username ${{ secrets.DOCKERHUB_USERNAME }} --password ${{ secrets.DOCKERHUB_TOKEN }}
+          TAG_VERSION=${{ steps.get_version.outputs.VERSION }}
+          docker push "${REGISTRY_HOSTNAME}/${IMAGE_NODE}:${TAG_VERSION}"
+          docker logout
```
[the remark this answers is not in the document]
Comment (miiu96) [reply .github/workflows/deploy-docker.yaml:35]: done

File: README.md [hunk 3]
```diff
@@ -101,3 +101,14 @@ The rest of endpoints remain the same.
...
+```
+ docker image build . -t chain-proxy-local -f ./docker/Dockerfile
+```
+
+## run proxy with docker
```
[the remark this answers is not in the document]
Comment (miiu96) [reply README.md:111]: fixed

**Review by AdoAdoAdo — APPROVED** [review]

**Review by liviuf-ancas — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -101,3 +101,14 @@ The rest of endpoints remain the same.
...
+
+
+## build docker image
+```
+ docker image build . -t chain-proxy-local -f ./docker/Dockerfile
+```
+
+## run proxy with docker
+```
+docker run -p 8080:8080 chain-proxy-local --log-level *:DEBUG
```
[the remark this answers is not in the document]
Comment (miiu96) [reply README.md:113]: added

**Review by ssd04 — APPROVED** [review]

**Review by liviuf-ancas — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Pull request merged**